devices.js

```diff
@@ -15202,6 +15202,37 @@ const devices = [
         description: 'E27 white and colour bulb',
         extend: preset.light_onoff_brightness_colortemp_colorxy,
     },
+
+    // Fantem
+    {
+        fingerprint: [{modelID: 'TS0202', manufacturerName: '_TZ3210_rxqls8v0'}],
+        model: 'ZB003-X',
+        vendor: 'Fantem',
+        description: '4 in 1 Multi Sensor',
+        fromZigbee: [
+            fz.battery,
+            fz.ignore_basic_report,
+            fz.illuminance,
+            fz.ZB003X,
+            fz.ZB003X_attr,
+            fz.ZB003X_occupancy,
+        ],
+        toZigbee: [tz.ZB003X],
+        exposes: [
+            e.occupancy(), e.battery(),
+            e.illuminance().withUnit('lx'),
+            e.temperature(), e.humidity(),
+            exposes.numeric('reporting_time', exposes.access.ALL),
+            exposes.numeric('temperature_calibration', exposes.access.ALL),
+            exposes.numeric('humidity_calibration', exposes.access.ALL),
+            exposes.numeric('illuminance_calibration', exposes.access.ALL),
+            exposes.binary('pir_enable', exposes.access.ALL, true, false),
+            exposes.binary('led_enable', exposes.access.ALL, true, false),
+            exposes.binary('reporting_enable', exposes.access.ALL, true, false),
+            exposes.enum('sensitivity', exposes.access.ALL, ['low', 'medium', 'high']),
+            exposes.enum('keep_time', exposes.access.ALL, ['0', '30', '60', '120', '240']),
```

Comment:
`exposes.access.ALL` should be `exposes.access.STATE_SET` since there is no convertGet